src/app.tsx

```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
     flexGrow: 1,
     padding: theme.spacing(3),
   },
+  close: {
+    padding: theme.spacing(0.5),
+  },
 }))
 /**
  * The main entry point of your app. You can start h@cking from here ;)
  */
 export const App: React.FunctionComponent = () => {
   const repo = useRepository()
   const [data, setData] = useState<Image[]>([])
-  const [uploaddata, setUploaddata] = useState<boolean>(false)
-  const [selectedimage, setSelectedimage] = React.useState<SelectedImage>({
-    imgIndex: 0,
-    imgPath: '',
-    imgTitle: '',
-    imgDescription: '',
-    imgAuthor: '',
-    imgAuthorAvatar: '',
-    imgCreationDate: '',
-    imgSize: '',
-    imgDownloadUrl: '',
-  })
+  const classes = useStyles()
+  const [isNotificationShown, ShowNotification] = React.useState<boolean>(false)
+  const [uploaddata, setUploaddata] = React.useState<boolean>(false)
+
+  /** Display the notificationbar about successful upload
+   * @param {boolean} switcher Should the notification shown or not
+   */
+  function DisplayNotification(switcher: boolean) {
+    ShowNotification(switcher)
+  }
   /**
-   * Sets the UploadData
+   * Hide the notificationbar
    */
-  function setUploaddataFunction() {
-    setUploaddata(true)
+  function CloseNotfication() {
+    ShowNotification(false)
   }
   /**
-   * Sets the Selected Image
-   * @param {number} imageIndex Seletected number's index.
+   * Sets the UploadData
+   * Determinate whatever an upload has accord or not
    */
-  function getSelectedImage(imageIndex: number) {
-    const selectedImage = data[imageIndex]
-    const avatarUser = selectedImage.CreatedBy as User
-    const avatarUserAvatarUrl = avatarUser.Avatar ? avatarUser.Avatar.Url : ''
-
-    setSelectedimage({
-      imgIndex: imageIndex,
-      imgPath: repo.configuration.repositoryUrl + selectedImage.Path,
-      imgTitle: selectedImage.DisplayName ? selectedImage.DisplayName : '',
-      imgDescription: selectedImage.Description ? selectedImage.Description : '',
-      imgAuthor: selectedImage.CreatedBy ? ((selectedImage.CreatedBy as User).FullName as string) : '',
-      imgAuthorAvatar: avatarUserAvatarUrl as string,
-      imgCreationDate: moment(new Date(selectedImage.CreationDate ? selectedImage.CreationDate : '')).format(
-        'YYYY-MM-DD HH:mm:ss',
-      ),
-      imgSize: `${(selectedImage.Size ? selectedImage.Size / 1024 / 1024 : 0).toFixed(2)} MB`,
-      imgDownloadUrl: selectedImage.Binary
-        ? repo.configuration.repositoryUrl + selectedImage.Binary.__mediaresource.media_src
-        : '',
-    })
+  function setUploaddataFunction() {
+    setUploaddata(true)
```

Comment:
You should check the repositry events we have. With that you can subscribe to changes in repository like content created, modified, etc... Checkout https://github.com/SenseNet/sn-client/blob/develop/packages/sn-hooks-react/src/context/current-children.tsx